src/yadmark/benchmark/external/apfelpy_utils.py

```diff
@@ -3,9 +3,9 @@
 from eko import basis_rotation as br
 
 # Q2 knots specs
-NQ = 250
+NQ = 1200
 QMin = 1
-QMax = 200
+QMax = 100
 
```

Comment:
Is there a reason to fix these values here, and not accepting them as input?

(moreover, if they are constants they have the wrong case)

Reply:
oh this was committed by mistake, no reason to change them.